moj-docker-deploy/files/ebs-tag.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+import boto
+import sys
+import boto.ec2
+import requests
+
```

Comment:
Since we're highstating when a machine boots how do we know we have these modules? Should we add a requires block in the pillar for the `boto` and `requests` pip?